Kinder/LikedItems.xaml.cs

```diff
@@ -32,9 +32,10 @@ public partial class LikedItems : Window
 
         //collection lists:
         private Lazy<List<User>> userList;
+        private Lazy<List<LikedItemsClass>> likedItemList;
 
         private List<Item> allItemList = new();
-        private List<LikedItemsClass> likedItemList = new();
+        //private List<LikedItemsClass> likedItemList = new();
```

Comment:
maybe delete it entirely, since it's commented and doesn't do anything

Reply:
done. thank you